src/main/java/org/broadinstitute/barclay/argparser/CommandLinePluginDescriptor.java

```diff
@@ -80,6 +80,7 @@
      * dynamically discoverable.
      * @return false if the class shouldn't be used; otherwise true
      */
+    // TODO: I don't think that this should be bounded, but I think that the default value should return c -> c instanceof getPluginClass()
     public Predicate<Class<?>> getClassFilter() { return c -> true;}
```

Comment:
Right - I think this should also not be bounded, and I like your proposed default implementation.